code-scanning/properties/rust-clippy.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "rust-clippy",
+    "creator": "The Rust programming language team",
```

Comment:
Since its the official language team, could we shorten creator to just `Rust` or `The Rust team`?

Reply:
Sounds good!